src/Directory.Version.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <VersionPrefix>3.0.41</VersionPrefix>
+    <VersionPrefix>3.1.0</VersionPrefix>
```

Comment:
Bumping minor I believe is correct under symver - but I think this repo just bumps patch - might wanna double check with Fabio